tests/test_cursor_namespace.py

```diff
@@ -69,5 +69,54 @@ def op():
         self._test_cursor_namespace(op, 'listIndexes')
 
 
+class TestKillCursorsNamespace(unittest.TestCase):
+    @classmethod
+    @unittest.skipUnless(version_tuple >= (3, 12, -1), 'Fixed in pymongo 3.12')
+    def setUpClass(cls):
+        cls.server = MockupDB(auto_ismaster={'maxWireVersion': 6})
+        cls.server.run()
+        cls.client = MongoClient(cls.server.uri)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls.client.close()
+        cls.server.stop()
+
+    def _test_killCursors_namespace(self, cursor_op, command):
+        with going(cursor_op):
+            request = self.server.receives(
+                **{command: 'collection', 'namespace': 'test'})
+            # Respond with a different namespace.
+            request.reply({'cursor': {
+                'firstBatch': [{'doc': 1}],
+                'id': 123,
+                'ns': 'different_db.different.coll'}})
+            # Client uses the namespace we returned for killCursors.
+            request = self.server.receives(**{
+                'killCursors': 'different.coll',
+                'cursors': [123],
+                '$db': 'different_db'})
+            request.reply({
```

Comment:
@ShaneHarvey should I just remove this entirely? We don't use it for anything...

Reply:
You definitely need to respond to the killCursors request or else the test will hang and timeout. Does this line cause a problem?

Reply:
Nevermind... Can't really entirely remove this as we need to have some response in place... Might as well let it be this.